src/makeRobotAccountant.js

```diff
@@ -6,7 +6,23 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return (param) => {
```

Comment:
```suggestion
  return (num) => {
```

`param` is too ambitious name for number 😄

Reply:
fixed